SPEC-annex-B-elf-embedding.md

```diff
@@ -0,0 +1,29 @@
+# Annex B
+
+Annex B contains a method of embedding OmniBOR ID into elf files.
+
+### Embedded Artifact Dependency Graph Identifier
+If an artifact contains an embedded artifact dependency graph identifier, then implementations MUST conform to the file format recommendations specified in this document.
+If an artifact does not contain an embedded artifact dependency graph identifier, then implementations SHOULD look for that information in alternate locations. Tools may store and retrieve such information in alternate formats.
+
+#### Gitoid persistence by a Build Tool in ELF Objects/Executables
+When persisting the gitoid to an ELF object or an ELF executable, the build tool should create a section ```.note.omnibor``` and place the gitoid in the descriptor field of the Note entry. This section must be of type SHT_NOTE and must have the attribute SHF_ALLOC. Multiple Note entries must be created, one for each gitoid type when multiple gitoid types are involved. Each Note entry must contain the following fields in the same order as given below:
```

Comment:
This should have a link to the ELF docs about how ELF sections work in general.

Reply:
Sec 2-4 in https://refspecs.linuxfoundation.org/elf/elfspec.pdf

Reply:
Fixed